### DISCUSSION TITLE
Image associated with Streamer Account

### DISCUSSION THREAD

**Discussion by Countryman98** [AzuraCast/AzuraCast#5304] [Ideas]
Hi Guys,

What's the chances of being able to add an Image to the Streamer/DJ Account section which can then be used (Maybe a new Widget)
a to Display on our website which Streamer/DJ/Show is connected live.

I know the dj/streamer name can be done somehow using the api but understand this does not display an image..
Also for many the api can be a little "above" our heads.. lol

**Comment by SpeedyCharly** [comment]
I suggest you put that in the suggestion section of the forum

**Reply by Countryman98** [reply]
> I suggest you put that in the suggestion section of the forum

I cant even seem to find the Suggestions section? This is Posted in ideas?

**Reply by SpeedyCharly** [reply]
FYI - To access the 'Suggestions' section simply start an new issue and you will have a list of the sections you can post to in the first page...

**Comment by BusterNeece** [comment]
This has already been suggested over at https://features.azuracast.com/, feel free to upvote the suggestion.

**Reply by Countryman98** [reply]
Done! Thanks
